tests/src/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -150,4 +153,40 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
 
             actionCode.delete
     }
+
+    /**
+    * Test an action that does not exceed the allowed number of open files.
+    */
+    it should "successfully invoke an action when it is within nofile limit" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "TestFileLimitGood-" + System.currentTimeMillis()
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, Some(openFileAction))
+            }
+
+            val run = wsk.action.invoke(name, Map("numFiles" -> (openFileLimit / 2).toJson))
+            withActivation(wsk.activation, run) {
+                activation =>
+                    activation.response.status shouldBe "success"
```

Comment:
this should be tighter: it should confirm the expected result `{ filesToOpen: numFiles, filesOpen: openFiles.length}`.